docs/using-yalc.md

```diff
@@ -0,0 +1,51 @@
+# Using `yalc` for Local SDK Testing
```

Comment:
Let's add a link to these doc to the main `CONTRIBUTING` doc.

Reply:
good point. [f6ba80f](https://github.com/getsentry/sentry-javascript/pull/7502/commits/f6ba80f5ae0809f1daf44408774b9ba72dbd7741)

(I think at some point we had a sentence about linking packages in there but not anymore. I added it back)